docs/_docs/pages.md

```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
 └── contact.html  # => http://example.com/contact.html
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. Unless there is a permalink entry in the page's front matter the same subfolders that are used to group your pages in your project's source will exist in the `_site` folder when your site builds. Each individual page is generated in said folders as file_name.html
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/doc1.html
```

Comment:
Does the filename get *downcased* at your end? Or is it just a typo?
On my Windows system, the **file path** is `_site\Documentation\Doc1.html`

Reply:
Just a typo, I will correct it now